app/gui/view/component-browser/component-list-panel/grid/src/entry.rs

```diff
@@ -196,11 +196,11 @@ impl DimmedGroups {
 #[allow(missing_docs)]
 #[derive(Clone, Debug, Default)]
 pub struct Params {
-    pub style:             Style,
-    pub grid_style:        GridStyle,
-    pub color_intensities: ColorIntensities,
-    pub group_colors:      GroupColors,
-    pub dimmed_groups:     DimmedGroups,
+    pub style:         Style,
+    pub grid_style:    GridStyle,
+    pub style_colors:  StyleColors,
```

Comment:
`StyleColors` is really not the best name. Do you have better ideas? This struct contains `Color` fields from the entry's stylesheet. `Colors` name is occupied by the struct containing `Lcha` samplers though

Reply:
What colors this struct contains? What is their purpose? Couldyou give an example of a few colors that are there?

Reply:
This struct contains color settings of the component group. Here is a definition:
```rust
pub struct StyleColors {
    #[theme_path = "entry_theme::text::color"]
    #[accessor = "Color::accessor"]
    pub text:            Color,
    #[theme_path = "entry_theme::background::color"]
    #[accessor = "Color::accessor"]
    pub background:      Color,
    #[theme_path = "entry_theme::highlight::hover::color"]
    #[accessor = "Color::accessor"]
    pub hover_highlight: Color,
    /// The more contrasting parts of the [icon](crate::icon::Any).
    #[theme_path = "entry_theme::icon::strong_color"]
    #[accessor = "Color::accessor"]
    pub icon_strong:     Color,
    /// The less contrasting parts of the [icon](crate::icon::Any).
    #[theme_path = "entry_theme::icon::weak_color"]
    #[accessor = "Color::accessor"]
    pub icon_weak:       Color,
    #[theme_path = "entry_theme::dimmed"]
    pub dimmed:          color::Rgba,
}
```
Later this struct is transformed into struct `Colors` containing FRP samplers for each color from the stylesheet 
```rust
pub struct Colors {
    pub background:          frp::Sampler<color::Lcha>,
    pub hover_highlight:    frp::Sampler<color::Lcha>,
    pub text:                       frp::Sampler<color::Lcha>,
    pub icon_strong:         frp::Sampler<color::Lcha>,
    pub icon_weak:           frp::Sampler<color::Lcha>,
    pub skip_animations: frp::Any,
}
```

So the flow is `stylesheet -> StyleColors -> <build FRP network for animations>  -> Color -> using it in FRP`

Reply:
I like that 👍🏻

Reply:
Suggestion applied.

Reply:
I like it as well!